packages/circuit-ui/components/NotificationModal/NotificationModal.tsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright 2019, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @jsx jsx */
+import { jsx, css, ClassNames } from '@emotion/core';
+import { ReactNode, MouseEvent, KeyboardEvent } from 'react';
+import ReactModal from 'react-modal';
+import { Theme } from '@sumup/design-tokens';
+
+import useClickHandler from '../../hooks/useClickHandler';
+import { ModalComponent, BaseModalProps } from '../ModalContext';
+import Image, { ImageProps } from '../Image';
+import Headline from '../Headline';
+import Body from '../Body';
+import Button, { ButtonProps } from '../Button';
+import ButtonGroup from '../ButtonGroup';
+import styled, { StyleProps } from '../../styles/styled';
+
+const TRANSITION_DURATION = 200;
+
+const imageStyles = ({ theme }: StyleProps) => css`
+  max-width: 232px;
+  height: 160px;
+  object-fit: cover;
```

Comment:
Reminder: this is unsupported in IE

Reply:
I know 😬 I'd argue that it's just a minor visual defect and doesn't prevent a user from using our product.

Do you think I should use a different, IE-safe approach?